vite.config.ts

```diff
@@ -35,9 +39,28 @@ export default defineConfig({
         plugins: [
           //  ['@babel/proposal-decorators', { legacy: true }],
           'formatjs',
+          'lodash',
           'preval',
+          'transform-react-remove-prop-types',
         ],
       },
     }),
   ],
+  test: {
+    environment: 'jsdom',
+    include: [
+      ...configDefaults.include,
+      '**/__tests__/**/*.{js,mjs,cjs,ts,mts,cts,jsx,tsx}',
+    ],
+    exclude: [
+      ...configDefaults.exclude,
+      '**/node_modules/**',
+      'vendor/**',
+      'config/**',
+      'log/**',
+      'public/**',
+      'tmp/**',
+    ],
+    globals: true,
```

Comment:
imo, moving away from globals would be recommended, though it does make writing tests a little more verbose, it plays better with ESM

Reply:
I agree, but it breaks things due to `testing-library` not running auto-cleanup, as indicated here: https://vitest.dev/guide/migration.html